docs/error-messages/compiler-errors-2/compiler-error-c2597.md

```diff
@@ -18,7 +19,9 @@ Possible causes:
 
 1. A member access operator refers to a nonmember function.
 
-1. The following sample generates C2597 and shows how to fix it:
+## Example
+
+1. The following example generates C2597 and shows how to fix it:
```

Comment:
A separate PR will be issued to remove the "1.", together with https://github.com/MicrosoftDocs/cpp-docs/pull/5583#discussion_r2231483942.